src/Model.cc

```diff
@@ -80,8 +81,15 @@ class sdf::Model::Implementation
   public: std::vector<Model> models;
 
   /// \brief The interface models specified in this model.
-  public: std::vector<std::pair<sdf::NestedInclude, sdf::InterfaceModelPtr>>
-              interfaceModels;
+  public: std::vector<std::pair<std::optional<sdf::NestedInclude>,
+          sdf::InterfaceModelConstPtr>> interfaceModels;
+
+  /// \brief The interface models specified in this model.
```

Comment:
update doc-string to distinguish from `interfaceModels`

Reply:
https://github.com/ignitionrobotics/sdformat/pull/768/commits/60708f0b108dcffe9021857b67250f0bc9c9cfda